hippy/frame.py

```diff
@@ -27,9 +30,33 @@ def match(self, interp, w_exc):
             return False
         return k_exc.is_subclass_of_class_or_intf_name(k_catch.name)
 
+    def py_match(self, frame, w_exc):
+        'Try to match w_exc to a Python exception.'
+        if not isinstance(w_exc, W_PyExceptionAdapter):
+            return False
+
+        interp = frame.interp
+        # try python builtin if there is no py parent
+        if frame.bytecode.py_scope is None:
+            try:
+                w_py_cls = interp.py_space.builtin.get(self.exc_class)
+            except OperationError:
+                return False
+        else:
+            ph_scope = PHPScope(interp, frame)
+            # w_py_cls = ph_scope.py_lookup_local_recurse(self.exc_class)
+            w_py_cls = frame.bytecode.py_scope.py_lookup_local(self.exc_class)
+            if w_py_cls is None:
+                w_py_cls = frame.bytecode.py_scope.py_lookup_global(self.exc_class)
+                if w_py_cls is None:
+                    return False
```

Comment:
This is more a general question, maybe for @vext01: is there a helper function in the hippy bridge somewhere for this kind of chained lookup? If not, should there be one?